### DISCUSSION TITLE
如何设置单个用户的存储容量呢？

### DISCUSSION THREAD

**Discussion by BigOldWei** [lsky-org/lsky-pro#426] [问答] [answered]
如题，我安装成功之后，默认容量只有500MB，我想扩大我的容量，怎么设置呢？翻遍了设置也没找到。

**Comment by penndu** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/46226528/169193422-0842bf80-e767-40b7-b14c-c82280f735d1.png)
